Core/Core/Api/GraphQL/Client.cs

```diff
@@ -188,7 +191,7 @@ public async Task<T> ExecuteGraphQLRequest<T>(GraphQLRequest request, Cancellati
     }
     // we log and wrap anything that is not a graphql exception.
     // this makes sure, that any graphql operation only throws SpeckleGraphQLExceptions
-    catch (Exception ex)
+    catch (Exception ex) when (!ex.IsFatal())
```

Comment:
Ahh I see this is the only change, I was curious about why we log for SpeckleGraphQLException<T> but not the others...  but probably beyoond the scope